addons/account/views/account_portal_templates.xml

```diff
@@ -132,6 +132,8 @@
                             <t t-call="portal.message_thread">
                                 <t t-set="token" t-value="invoice.access_token"/>
                                 <t t-set="object" t-value="invoice"/>
+                                <t t-set="pid" t-value="pid"/>
```

Comment:
Is it intended to name it pid and not partner_id ?

Reply:
it was to reduce the number of character in the url. but as we will now encode the url parameters, we can enventually rename it to partner_id !?